deploy/base/model.yaml

```diff
@@ -1,2 +1,4 @@
 small: embeddings_small_2023_05_24_23_17_03.csv
 medium: embeddings_medium_2023_05_24_22_15_24.csv
+finetuned_alternatives_model: ft_alt_all-MiniLM-L6-v2_v20230623171244
+finetuned_final_model: ft_final_ft_alt_all-MiniLM-L6-v2_v20230623171244_v20230623171328
```

Comment:
Should the files end with `.csv`?

Reply:
These are not csv files. In fact, the model is a directory.